app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileFragment.java

```diff
@@ -1,31 +1,88 @@
 package com.cmput301f20t21.bookfriends.ui.profile;
 
+
+import androidx.annotation.NonNull;
 import androidx.lifecycle.ViewModelProvider;
 
+import android.content.Intent;
 import android.os.Bundle;
 
-import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
 import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
 
 import com.cmput301f20t21.bookfriends.R;
+import com.cmput301f20t21.bookfriends.ui.login.LoginActivity;
 
-public class ProfileFragment extends Fragment {
+
+public class ProfileFragment extends Fragment implements ProfileEditDialog.ProfileEditListener {
 
     private ProfileViewModel mViewModel;
+    public TextView Email;
+    public TextView Phone;
```

Comment:
first letter of each variable should be lowercase unless it is a class name or declared static final
and these attributes should be private